test/Ocelot.UnitTests/Logging/OcelotLoggerTests.cs

```diff
@@ -160,42 +149,31 @@ public void If_minimum_log_level_set_to_none_then_log_method_is_never_called()
         var currentLogger = new OcelotLogger(mockedILogger.Object, repo.Object);
 
         currentLogger.LogDebug(() => $"a message from {_a} to {_b}");
+        var expected = "requestId: no request id, previousRequestId: no previous request id, message: 'a message from tom to laura'";
```

Comment:
should be requestId: No RequestId, previousRequestId: No PreviousRequestId, otherwise you are not testing the log level...

Reply:
My fault... :cry:

Reply:
Yes, because the verify method is checking if the string has been used for logging...